var/spack/repos/builtin/packages/py-futures/package.py

```diff
@@ -12,6 +12,7 @@ class PyFutures(PythonPackage):
     homepage = "https://pypi.python.org/pypi/futures"
     url      = "https://pypi.io/packages/source/f/futures/futures-3.0.5.tar.gz"
 
+    version('3.2.0', sha256='9ec02aa7d674acb8618afb127e27fde7fc68994c0437ad759fa094a574adb265')
     version('3.0.5', sha256='0542525145d5afc984c88f914a0c85c77527f65946617edb5274f72406f981df')
 
     depends_on('py-setuptools', type='build')
```

Comment:
Can you add:
```python
depends_on('python@2.6:2.8', type=('build', 'run'))
```
It looks like this package won't build for Python 3 (and shouldn't be used anyway).